openspec/changes/add-tantivy-search/tasks.md

```
@@ -0,0 +1,44 @@
+# 实施任务清单
+
+## 1. 依赖设置
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)